README.md

```diff
@@ -434,14 +438,14 @@ to work, we needed to complete the following steps:
 - In the top-level component (`App.js` in this case), create our "parent" routes
   and render `<MoviesPage>`
 - In `MoviesPage.js`, render `<MoviesList>`
-- In `MoviesList.js`, iterate through the `movies` object and create a dynamic
+- In `MoviesList.js`, iterate through the `movies` array and create a dynamic
   `Link` for each movie using its id
 - Back in `MoviesPage.js`, import `useRouteMatch` and create the child route by
   combining the current url with the `:movie_id` parameter; inside the child
-  route, render `<MovieShow>`, passing the `movies` object as props
-- In `MovieShow.js`, import `useParams`; use the `:movie_id` from the params
-  object to access the correct movie from the `movies` object and display it on
-  the page
+  route, render `<MovieShow>`, passing the `movies` array as props
+- In `MovieShow.js`, import `useParams`; use the `:movieId` from the params
+  object to find the correct movie from the `movies` array and display it on the
+  page
 
 In setting up our nested routes, we made use of two hooks provided by React
 Router: `useRouteMatch` and `useParams`. The first is used to retrieve the URL
```

Comment:
There are two occurrences of ':movie_id' in the Conclusion that should be updated to ':movieId'

Reply:
Fixed!